react/CodeBlock.js

```diff
@@ -0,0 +1,81 @@
+const React = require('react')
+const ReactDOM = require('react-dom')
+const { Prism } = require('react-syntax-highlighter')
```

Comment:
We use http://ghub.io/rehype-highlight (via `docs/render-content` via [hubdown](https://github.com/electron/hubdown#plugins)) for syntax highlighting everywhere else. If we use Prism here, we're probably going to get different highlighting?

Reply:
Yeah we could! My work was solely to do a proof of concept to make sure we could do whatever we want. I'm not actually recommended the use of any of these specific react components. Just making sure we can render any react component and have it be functional.